src/Extensions/ServiceFabric.Services/OmexServiceRuntime.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Omex.Extensions.Startup;
+using Microsoft.ServiceFabric.Services.Runtime;
+
+namespace Microsoft.Omex.Extensions.ServiceFabric.Services
+{
+	/// <summary> Static class that provides methods to register reliable services with </summary>
+	public class OmexServiceRuntime : OmexApplicationStartup<OmexServiceFabricContext>
+	{
+		/// <inheritdoc/>
+		public OmexServiceRuntime(string? serviceTypeName = null) : base(serviceTypeName) { }
+
+
+		/// <summary>Registers a reliable stateful service with Service Fabric runtime</summary>
+		/// <param name="serviceTypeName">The service type name as provied in service manifest</param>
+		/// <param name="typeRegestration"> Method to register aditional types in DI </param>
```

Comment:
>  [](start = 37, length = 1)

nit: space

Reply:
same in the method below

---
In reply to: [362439129](https://github.com/microsoft/Omex/pull/153#discussion_r362439129) [](ancestors = 362439129)

Reply:
Class deleted